extensions/docbook-snippets/vsc-extension-quickstart.md

```diff
@@ -0,0 +1,23 @@
+# Welcome to your VS Code Extension
+
+## What's in the folder
+
+* This folder contains all of the files necessary for your extension.
```

Comment:
Who is the target user? The developer writing this extension or the user using it?

Maybe add your target group.